src/components/CommentList.jsx

```diff
@@ -0,0 +1,34 @@
+import fetchComments from "../utils/fetchComments";
+import CommentCard from "./CommentCard";
+import { useEffect, useState } from "react";
+
+function CommentList(reviewId) {
+    const [comments, setComments] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+
+    useEffect(() => {
+        fetchComments(reviewId.reviewId)
+        .then((comments) => {
+            setComments(comments)
+        })
+        .then(() => {
+            setIsLoading(false)
+        })
+    }, [])
+
+    if (isLoading) {
+        return <p>Loading comments...</p>
+    } else {
+        return (
+            <div>
+                <ul className="comments-list">
```

Comment:
What happens if a review has 0 comments? Can we add something to handle that case